### PR TITLE
update latest oC version

### PR DESCRIPTION
**Pull request by voroyam**
Now the latest one is 10.5.0

### CODE REVIEW

File: modules/admin_manual/pages/_partials/maintenance/major_release_note.adoc [hunk 1]
```diff
@@ -11,9 +11,17 @@ Here are some examples:
...
 |Version
-|Can Upgrade to 10.3.2?
+|Can Upgrade to 10.5.0?
 |Requirements
 
+|10.4.1
+|Yes
+|
+
+|10.3.1
```
Comment (phil-davis) [inline modules/admin_manual/pages/_partials/maintenance/major_release_note.adoc:21]:
```suggestion
|10.3.2
```

**Comment by phil-davis** [comment]
This should be backported to 10.5 branch.
Then I guess 10.4 branch can be checked, and adjusted appropriately if necessary.

File: modules/admin_manual/pages/_partials/maintenance/major_release_note.adoc (hunk 1 above)
Comment (voroyam) [reply modules/admin_manual/pages/_partials/maintenance/major_release_note.adoc:21]: Can you explain why you want that change? :D

simply suggesting a change without explanation is weird.

Comment (phil-davis) [reply modules/admin_manual/pages/_partials/maintenance/major_release_note.adoc:21]: The other examples in the list have the latest patch release as the example.

**Comment by phil-davis** [comment]
PR #2707 has a backport of what happened in master. Maybe it would be cleaner and more traceable to merge that?

**Pull request closed without merging**